andela/eventx
# #105616162 Search with Map

#### What does this PR do?
Enables users to search event with the map
#### Description of Task to be completed?
Add api key
Get and convert events to json
Render events on map
#### How should this be manually tested?
Visit the event page
Drap the map to the desired location, available events should be rendered by default
You could also type in the location on the search form, and see the events for the specified location
#### Screenshots
![image](https://cloud.githubusercontent.com/assets/17007838/15513321/07644bcc-21dc-11e6-8a1c-0fe03ab60740.png)
